code/data_structures/src/stack/stackUsingLinkedList/stackUsingLinkedList.cpp

```diff
@@ -0,0 +1,90 @@
+#include<iostream>
+#include<malloc.h>
+using namespace std;
+struct node
+{
+    int data;
+    struct node *next;
+};
+void Traverse_stack(struct node* top)
+{
+    struct node* temp=top;
+    cout<<"Current Stack->\n";
+    while(temp!=NULL)
+    {
+    cout<<temp->data<<"\n";
+    temp=temp->next;
+    }
+}
+struct node * push(struct node * top,int data)
+{
+    struct node * temp;
+    temp=(struct node*)malloc(sizeof(struct node));
```

Comment:
Use new and delete in C++.

Reply:
ha i got it.. sir I think I should commit the same code in C language also and also make changes and submit the modified code in c++